### PR TITLE
Give a try at #10 about Python 2 vs. 3.

### PR DESCRIPTION
**Pull request by pauleveritt**
Give a try at #10 about Python 2 vs. 3.

### CODE REVIEW

File: src/templates/index.html [hunk 1]
```diff
@@ -58,10 +58,13 @@ <h3 class="title">Hello World App</h3>
         </div>
         <div class="col-md-6">
           <h3 class="title">Easy To Start</h3>
-          <p>Let's create a minimal web app in 5 minutes using Python 3.</p>
+          <p>Let's create a minimal web app in 5 minutes.</p>
           <h4>1. Install Python</h4>
-          <p>Install the latest Python 3 from <a href="https://www.python.org/downloads/">Python.org</a>.</p>
+          <p>Install the latest Python from <a
```
Comment (stevepiercy) [inline src/templates/index.html]: Install the latest Python 2 or 3 from

File: src/templates/index.html [hunk 2]
```diff
@@ -58,10 +58,13 @@ <h3 class="title">Hello World App</h3>
         </div>
         <div class="col-md-6">
           <h3 class="title">Easy To Start</h3>
-          <p>Let's create a minimal web app in 5 minutes using Python 3.</p>
+          <p>Let's create a minimal web app in 5 minutes.</p>
           <h4>1. Install Python</h4>
-          <p>Install the latest Python 3 from <a href="https://www.python.org/downloads/">Python.org</a>.</p>
+          <p>Install the latest Python from <a
+                  href="https://www.python.org/downloads/">Python.org</a>.
+          Works fine under Python 2 or 3.</p>
```
Comment (stevepiercy) [inline src/templates/index.html]: Pyramid runs on both versions.

Comment (pauleveritt) [inline src/templates/index.html]: I was trying to be very, very terse on the wording in that section.

—Paul

Comment (stevepiercy) [inline src/templates/index.html]: Maybe just remove it? The previous amended statement implies the same thing anyway.

Comment (pauleveritt) [inline src/templates/index.html]: The Python 3 part under "Install Pyramid" is due to the presence of pyvenv in the instructions.

We're over-talking this. Just merge whatever you'd like.

**Pull request merged**